pkg/api/v1/types.go

```diff
@@ -354,6 +371,11 @@ const (
 	// used for PersistentVolumes where the bound PersistentVolumeClaim was deleted
 	// released volumes must be recycled before becoming available again
 	VolumeReleased PersistentVolumePhase = "Released"
+	// used for PersistentVolumes that have been recycled and can be made available
+	// again to new PersistentVolumeClaims
+	VolumeRecycled PersistentVolumePhase = "Recycled"
```

Comment:
For future reference:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/principles.md#control-logic